DummyServer/ClientApp/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "dummyserver",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "build:ssr": "ng run DummyServer:server:dev",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "8.2.12",
+    "@angular/common": "8.2.12",
+    "@angular/compiler": "8.2.12",
+    "@angular/core": "8.2.12",
```

Comment:
Any particular reason for not using Angular 10 since this is currently the latest version?

Reply:
This was the most recent version when I started to build the DummyServer. Updating to 10 doesn't work without extra work, so I created #188